### PR TITLE
Awesome books: style core elements

### PR DESCRIPTION
**Pull request by oliverSCZ**
I did the following:

- l added CSS styles to the Awesome books application.
- Maintain all of the original functionality of the application.
-  Opened this pull request as instructed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,33 +13,81 @@
...
+            <li class="contact-item">
+              Our address: 3448 Willis Ave, Perry, IA 50220, United States
+            </li>
+          </ul>
+        </section>
+      </main>
+      <footer class="footer">
+        <p class="copyright footer-item">&copy; 2021</p>
+        <p class="author footer-item">by Oliver Coimbra</p>
+      </footer>
```
Comment (mikethreels) [inline index.html:82]: - [ ] great job making the changes and keeping the functionality 👍 the footer is floating on top of the other elements, which makes the application look a little unorganized. please change the application so the footer is always below all the other elements.
127 0 0 1_5500_awesome-books_index html_

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Oliver, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -1,37 +1,198 @@
...
+
+.nav-item:last-child {
+  border-right: none;
+}
+
+.hero {
+  width: 95%;
+  height: 87vh;
+  padding-top: 3rem;
+}
```
Comment (TedLivist) [inline style.css]: - [x] You still have not resolved the footer issue as highlighted by the previous reviewer (see image below). All through your CSS you are using **`vh`** unit for your heights. But technically, the total viewport-height is 100. When all the **`vh`** values you have are added the result is more than 100.

  Here is a suggestion: Remove the line **`height: 87vh;`** here. It should correct your styling issue.

oliver-foot

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @oliverSCZ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, the team @oliverSCZ  👋🏾 👋🏾 congratulations on making it so far on this milestone.

## Optional suggestions

- Please work on the height of the page, the contact form is being disappeared, please fix it.
- The menu should be clickable and redirect to the page we need.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

**Pull request merged**